lib/ansible/modules/cloud/amazon/ec2_asg.py

```diff
@@ -82,7 +82,10 @@
       - Desired number of instances in group, if unspecified then the current group value will be used.
   replace_all_instances:
     description:
-      - In a rolling fashion, replace all instances with an old launch configuration with one from the current launch configuration.
+      - In a rolling fashion, replace all instances with an old launch configuration with one from the current launch configuration. 
```

Comment:
"replace all instances with an old launch configuration" is confusing (it was confusing before this change)

Might be better as "replace all instances from an old launch configuration"